web/main/models.py

```diff
@@ -2060,6 +2060,15 @@ def contents(self):
         res = ContentNode.objects.filter(**filter_map).exclude(id=self.id)
         return res
 
+    @property
+    # gets the next node ordinals
+    def next_node(self) -> tuple(int, ...):
```

Comment:
```suggestion
    def next_node(self) -> Optional[Tuple(int, ...)]:
```

Need `from typing import Optional, Tuple` at the top too

Reply:
hmm seems to be still failing.. 🥲 what does the `-> Optional[Tuple(int, ...)]:` part do??

Reply:
oh i'm clowning, it should be

`    def next_node(self) -> Optional[Tuple[int, ...]]:`

It's a type hint that lets mypy know what type of object to expect when next_node is returned

Reply:
gotcha! Thanks!